test/TestHelper.cpp

```diff
@@ -119,6 +119,32 @@ ImportTest::ImportTest(json_spirit::mObject& _o, bool isFiller):
 	}
 }
 
+json_spirit::mObject& ImportTest::makeAllFieldsHex(json_spirit::mObject& _o)
+{
+	static const std::string hashes[] = {"bloom" , "coinbase", "hash", "mixHash", "parentHash", "receiptTrie",
```

Comment:
Can you use `static const set<string> hashes{"bloom", ..., "secretKey"}` here and
```
if (hashes.count(key))
  continue;
```
below?